lms/djangoapps/learner_dashboard/test_serializers.py

```diff
@@ -69,7 +69,9 @@ def test_linkage(self):
         with mock.patch(
             "lms.djangoapps.learner_dashboard.serializers.PlatformSettingsSerializer.to_representation"
         ) as mock_platform_settings_serializer:
-            mock_platform_settings_serializer.return_value = mock_platform_settings_serializer
+            mock_platform_settings_serializer.return_value = (
```

Comment:
style change for consistency?

Reply:
I have a habit of running `black` on all my new commits. I think this one fell through the cracks from previous work but removing as out-of-scope for this PR.